1526999244518	Sync.LogManager	DEBUG	Flushing file log
1526999244532	Sync.LogManager	DEBUG	Log cleanup threshold time: 1526135244532
1526999244617	Sync.LogManager	DEBUG	Done deleting files.
1526999245484	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1526999347120	Sync.SyncScheduler	DEBUG	Network link status changed to "down". Offline?: false
1526999347128	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1526999347139	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=schedule)
1526999348123	Sync.SyncScheduler	DEBUG	Network link status changed to "changed". Offline?: false
1526999350119	Sync.SyncScheduler	DEBUG	Network link status changed to "up". Offline?: false
1526999350119	Sync.SyncScheduler	DEBUG	Network link looks up. Syncing.
1526999350144	Sync.Service	DEBUG	User-Agent: Firefox/60.0 (Linux x86_64) FxSync/1.62.0.20180509234620.desktop
1526999350145	Sync.Service	INFO	Starting sync at 2018-05-22 21:29:10 in browser session 8IYZaSGUJ7Gm
1526999350146	Sync.SyncScheduler	DEBUG	Clearing sync triggers and the global score.
1526999350161	Sync.Status	INFO	Resetting Status.
1526999350161	Sync.Status	DEBUG	Status.service: error.sync.failed => success.status_ok
1526999350197	Sync.Resource	WARN	GET request to https://sync-600-us-west-2.sync.services.mozilla.com/1.5/92728811/info/collections failed: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526999350198	Sync.Status	DEBUG	Status.sync: success.sync => error.login.reason.network
1526999350198	Sync.Status	DEBUG	Status.service: success.status_ok => error.sync.failed
1526999350198	Sync.ErrorHandler	ERROR	Sync encountered an error: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526999350200	Sync.SyncScheduler	DEBUG	Next sync in 600000 ms. (why=reschedule)
1526999350210	Sync.Service	DEBUG	Exception calling anonymous function: [Exception... "The lookup of the hostname failed"  nsresult: "0x804b001e (NS_ERROR_UNKNOWN_HOST)"  location: "<unknown>"  data: no] No traceback available
1526999350211	Sync.ErrorHandler	DEBUG	Addons installed: 8
1526999350211	Sync.ErrorHandler	DEBUG	 - Firefox Multi-Account Containers, version 6.0.0, id @testpilot-containers
1526999350211	Sync.ErrorHandler	DEBUG	 - GNOME Shell integration, version 10.1, id chrome-gnome-shell@gnome.org
1526999350211	Sync.ErrorHandler	DEBUG	 - Web Extension for Shaarli, version 2.0.0, id shaarli@imirhil.fr
1526999350212	Sync.ErrorHandler	DEBUG	 - DuckDuckGo Privacy Essentials, version 2018.5.1, id jid1-ZAdIEUB7XOzOJw@jetpack
1526999350213	Sync.ErrorHandler	DEBUG	 - FoxyProxy Standard, version 6.2, id foxyproxy@eric.h.jung
1526999350213	Sync.ErrorHandler	DEBUG	 - Zotero Connector, version 5.0.39, id zotero@chnm.gmu.edu
1526999350213	Sync.ErrorHandler	DEBUG	 - Sync Tab Groups, version 0.6.9, id sync-tab-groups@eric.masseran
1526999350213	Sync.ErrorHandler	DEBUG	 - uBlock Origin, version 1.16.6, id uBlock0@raymondhill.net
